web-console/knikubevirt/openshift-vms-overview/openshift-vms-overview.md

```diff
@@ -0,0 +1,29 @@
+# OpenShift VMs Overview
+
+The purpose of this PR is to explore what Virtual Machines could look like in the Overview and Resources sections of Openshift/CNV.
+This is the first pass so really just trying to understand exactly what needs to be surfaced there.
+
+Through the Overview page the user can see the VM’s based on the application it is associated with via it’s label.
+
+![list of VMs based on app with label](Openshift-overview-virtual-machine.png)
```

Comment:
I think there are some updates needed to align with the latest and greatest layout...

1) Breadcrumbs, project name, and actions dropdown at the top should replace project selection.
2) This should be a tabbed layout since it's a project details page with the "Workloads" tab selection.
3) Group by and filter placement should be updated.
4) Looks like the donut chart for the pod isn't there already. @beanh66 is this coming back?
5) Notice a few updates needed to the navigation.

Here is a screenshot from a sample environment:
<img width="1680" alt="Workloads" src="https://user-images.githubusercontent.com/208455/60289548-ad5c8d00-98e4-11e9-9955-477fe9728933.png">

Reply:
Hi @lizsurette & thanks for the comments. 
I'm not exactly sure where should the “group by” be placed in a table according to PF4. 
Thanks.